scripts/sanitycheck

```diff
@@ -176,11 +176,17 @@ from collections import OrderedDict
 from itertools import islice
 import yaml
 
+import logging
+log_format = "%(levelname)s %(name)s::%(module)s.%(funcName)s():%(lineno)d: %(message)s"
```

Comment:
This sets the policy of how logging is to be reported; the modules/libraries log and the output is done based on what this policy says.